### PR TITLE
Remove a reference to cider-inspect-read-and-inspect.

### PR DESCRIPTION
**Pull request by cskksc**
It fixes the bytecomp travis builds.

### CODE REVIEW

File: cider-inspector.el [hunk 1]
```diff
@@ -112,7 +112,7 @@ With a second prefix argument it prompts for an expression to eval and inspect."
   (pcase arg
     (1 (cider-inspect-last-sexp))
     (4 (cider-inspect-defun-at-point))
-    (16 (cider-inspect-read-and-inspect))))
+    (16 (call-interactively #'cider-inspect-expr))))
```
Comment (bbatsov) [inline cider-inspector.el]: This change seems unrelated to the primary purpose of the commit. It's also unclear to me what exactly do we gain from this change.

**Comment by bbatsov** [comment]
Commit message titles should not end with `.`.

File: cider-inspector.el (hunk 1 above)
Comment (Malabarba) [inline cider-inspector.el]: This is necessary because I've marked that function as obsolete.

Comment (cskksc) [inline cider-inspector.el]: This fixes the travis bytecomp builds, https://travis-ci.org/clojure-emacs/cider/jobs/124647689#L969 and https://travis-ci.org/clojure-emacs/cider/jobs/124622642#L1065

Comment (cskksc) [inline cider-inspector.el]: Sorry about the commit message thing. I seem to be making this mistake repeatedly. Won't happen again. :-)
